modules/cnf-deprovisioning-clusters.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// Epic CNF-9657 (TELCODOCS-1540) (4.16)
+// * edge_computing/cnf-understanding-siteconfig-operator.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cnf-deprovisioning-clusters_{context}"]
+= Deprovisioning {sno} clusters with the {sco}
+
+Delete your clusters with the {sco}.
+
+.Prerequisites
+
+* You have deployed your clusters with the {sco} by using the default cluster templates.
+
+.Procedure
+
+* Delete the `ClusterInstance` CR by running the following command:
++
+[source,terminal]
+----
+$ oc delete clusterinstance <cluster_name> -n <target_namespace> 
+----
+
+.Verification
+
+* Verify that the deletion was successful by running the following command:
++
+--
+[source,terminal]
+----
+$ oc get clusterinstance <cluster_name> -n <target_namespace>
+----
+
+.Example output
+[source,terminal]
+----
+Error from server (NotFound): the server could not find the requested resource (delete clusterinstances.siteconfig.open-cluster-management.io <cluster_name>)
```

Comment:
This doesn't look right to me. Maybe you had a typo when you ran the command? I say that because the word `delete` wouldn't be in the response to a `oc get`

Reply:
Good catch @carbonin! I provided the incorrect output to @amolnar-rh. Please change to:
```
Error from server (NotFound): clusterinstances.siteconfig.open-cluster-management.io "<cluster_name>" not found
```